setup.py

```diff
@@ -69,10 +69,9 @@ def find_version(*file_paths):
 
 version_string = "{}{}".format(sys.version_info[0], sys.version_info[1])
 
-INSTALL_REQUIRES.extend(["torch", "torchvision"])
+INSTALL_REQUIRES.extend("torch")
 
 TORCH_VERSION = "1.7.0"
-TORCHVISION_VERSION = "0.8.1"
 CUDA_VERSION = "102"
 IS_CUDA_VER_DEFAULT_FOR_CURRENT_TORCH_VER = True
 
```

Comment:
Why we need TORCHVISION_SOURCE_URL_TEMPLATE in 80 line if it is not used any more?

Reply:
you are right; removed